MAUI/LinearGauge/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: About .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control, its features, and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# .NET MAUI Linear Gauge (SfLinearGauge) Overview
+
+Syncfusion .NET MAUI Linear Gauge is a data visualization control to display data on a linear scale. Use this control to craft high-quality mobile app user interfaces.
+
+![Overview .NET MAUI linear gauge](images/basic_elements.PNG)
+
+## Key Features
+
+* **Orientation** - The Linear Gauge can be set to vertical or horizontal orientation.
+* **Scale** - The scale is a set of values can be plotted. An scale can be customized by changing the thickness and edge styles and you can also inverse the scale.
```

Comment:
The scale is a set of values that can be plotted. You can customize a scale by changing the thickness and edge styles, and you can also inverse the scale.